### ISSUE TITLE
[Accessibility][Screen reader-Bread Crumb]: In Scan mode user is not able to close the 'More options menu' using 'Esc key'

### ISSUE THREAD

**Issue by dollykumari5** [OfficeDev/office-ui-fabric-react#9689]
Detailed description of this issue can be found here [Sharepoint](https://microsoft-my.sharepoint.com/:x:/p/v-amaver/EQDF8CVEaSVLgoImESsriiUBPXsgaMRfBvMKy_67oM_1Bw?e=yQiu5S)

Component:Breadcrumb

**Comment by khmakoto** [comment]
Hi @dollykumari5, thanks for filing this! I'll investigate the issue to see if this is something on our part, on Narrator's part or by design in scan mode given how it breaks from author's intent in dictation and navigation.

**Comment by khmakoto** [comment]
Just wanted to give an update here that I'm still waiting for a response from the Narrator team regarding this issue.

**Comment by dzearing** [comment]
@khmakoto Any update?

**Comment by kenotron** [comment]
In Scan mode, there is no documented way to "ESC" command:
https://support.microsoft.com/en-us/help/22809/windows-10-narrator-using-scan-mode

When the OS is in Scan Mode, the key events no longer happen the way that is normally expected. So, in other words, there is no way to control what escape does during Scan Mode. We therefore cannot affect this in anyway.
